File: app/src/main/java/br/com/doghero/dhproject/images/ImageHelper.java
package br.com.doghero.dhproject.images;

import android.content.Context;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

public class ImageHelper {
    public static void loadImage(Context context, String imageUrl, int placeHolderResourceId, ImageView imageView) {
        Picasso.with(context)
                .load(imageUrl)
                .placeholder(placeHolderResourceId)
                .into(imageView);
    }
}

File: app/src/main/java/br/com/doghero/dhproject/helper/ImageHelper.kt
package br.com.doghero.dhproject.helper

import android.content.Context
import android.widget.ImageView
import com.squareup.picasso.Picasso

object ImageHelper {

    private fun loadImage(context: Context): Picasso {
        return Picasso.with(context)
    }

    fun loadImage(context: Context, imageUrl: String, placeHolderResourceId: Int, imageView: ImageView) {
        this.loadImage(context)
                .load(imageUrl)
                .placeholder(placeHolderResourceId)
                .into(imageView)
    }

    fun loadImage(context: Context, imageId: Int, placeHolderResourceId: Int, imageView: ImageView) {
        this.loadImage(context)
                .load(imageId)
                .placeholder(placeHolderResourceId)
                .into(imageView)
    }

}